.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text eol=crlf
```

Comment:
According to an Stack overflow the first one defines Server side and the second one client side.